tests/tools/oesign-engine/test-enclave/enclave/test-enclave-kss.conf

```diff
@@ -0,0 +1,13 @@
+# Copyright (c) Open Enclave SDK contributors.
+# Licensed under the MIT License.
+
+# Enclave settings:
+Debug=1
+Kss=1
+NumHeapPages=1024
+NumStackPages=1024
+NumTCS=1
+ProductID=1
+SecurityVersion=1
+IsvFamilyID=7cc330ff-c8cf-49d4-92ef-e1674794f820
```

Comment:
It would be great to add more test cases to uuid_from_string() especially if IsvFamilyID or IsvExtProductID are not in correct UUID format. Since these values are specified externally, it's important that we can handle all error cases.

Reply:
Update by adding longer or shorted ID test cases